mlrun/model_monitoring/stores/kv_model_endpoint_store.py

```diff
@@ -512,3 +535,15 @@ def validate_old_schema_fields(endpoint: dict):
             ] = endpoint[
                 mlrun.common.schemas.model_monitoring.EventFieldType.ENDPOINT_ID
             ]
+
+    @staticmethod
+    def _encode_decode_field(
+        field: typing.Union[str, bytes], encode: bool = True
```

Comment:
No reason to provide default to `encode` param (they are symetric).
Also why provide _endode_field() and _decode_field(), what do we gain in provding both in one function?